stable/xray/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+name: xray
+version: 0.3.1
```

Comment:
This chart is already in incubator with a version of `0.3.1`. There is a process for promoting a chart from incubator to stable ([documented here](https://github.com/helm/charts/blob/master/PROCESSES.md#promoting-a-chart-from-incubator-to-stable)). Can you follow that and make sure the version in incremented?

The process is designed to help inform existing users figure out the change.

Reply:
@mattfarina Here is the PR for depreciating incubator chart https://github.com/helm/charts/pull/7156